app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionsViewModel.java

```diff
@@ -56,6 +60,8 @@ public void destroy() {
                     this.rooms = extractRooms(sessions);
                     this.stimes = extractStimes(sessions);
 
+                    notifyPropertyChanged(BR.loadingVisibility);
```

Comment:
I found an issue to update visibility in doOnSuccess/doOnError.
doOnSuccess and doOnError run on not-UI thread therefore notifyPropertyChanged causes CalledFromWrongThreadException.
Of course I can notify with Handler... what should I do?
Using Handler just for notifyPropertyChange may be too much.